src/interface/web/app/home/page.tsx

```diff
@@ -0,0 +1,463 @@
+            } catch (error) {
+                console.error("Error starting a conversation with the agent:", error);
+            }
+        };
+    }
+
+// Helper function to convert Tailwind color class to actual color value to prevent default border changes from being applied
+function tailwindColorToHex(colorClass: string): string {
+    const colorMap: { [key: string]: string } = {
+        'border-red-500': '#ef4444',
+        'border-blue-500': '#3b82f6',
+        'border-green-500': '#22c55e',
+        'border-yellow-500': '#eab308',
+        'border-purple-500': '#a855f7',
+        'border-pink-500': '#ec4899',
+        'border-indigo-500': '#6366f1',
+        'border-gray-500': '#6b7280',
+        'border-orange-500': '#f97316',
+    };
+    return colorMap[colorClass] || '#000000'; // Default to black if color not found
+}
+
+function highlightHandler(slug: string) {
+    // find buttons with AGENT in their class name
+    const buttons = document.getElementsByClassName("AGENT");
+
+    // find the color of the icon inside the agent
+    const color = agents.find(agent => agent.slug === slug)?.color;
+    const color_class = convertColorToClass(color || 'gray').split(' ')[0];
+
+    // replace 'bg' with 'border' to get the tailwind border color
+    const border_color = color_class.replace('bg', 'border');
+
+    // Convert Tailwind color class to actual color value
+    const hexColor = tailwindColorToHex(border_color);
+
+    for (let i = 0; i < buttons.length; i++) {
+        const button = buttons[i] as HTMLElement;
+        if (button.classList.contains(slug)) {
+            // Set the border color directly using the hex value
+            button.style.borderColor = hexColor;
+            button.style.borderWidth = '1px';
+            button.style.borderStyle = 'solid';
+        } else {
+            // Remove border from other buttons
+            button.style.borderColor = '';
+            button.style.borderWidth = '';
+            button.style.borderStyle = '';
+            button.style.boxShadow = '';
+
+            // also remove any -500 ending class (handles start case where khoj is selected by default at the start)
+            for (let j = 0; j < button.classList.length; j++) {
+                if (button.classList[j].endsWith('-500')) {
+                    button.classList.remove(button.classList[j]);
+                }
+            }
+        }
+    }
+}
+
+    return (
+        <div>
+        <div className="w-full text-center">
+        <div className="items-center">
+            <h1 className="text-center pb-6">What would you like to do?</h1>
+        </div>
+        <div className="flex pb-6 ms-10 gap-2">
+            {icons.map((icon, index) => (
+                <a className={`AGENT ${agents[index].slug} no-underline w-200 flex pl-3 pt-1 pb-1 border ${agents[index].slug === "khoj" ? "border-orange-500" : "border-stone-100"} rounded-md shadow-sm`} onClick={handleAgentsClick(agents[index].slug)}>
```

Comment:
nit: classnames should be camel case if coming from the `module.css`, or hyphenated in the case of tailwind. 
 
```suggestion
                <a className={`agent ${agents[index].slug} no-underline w-200 flex pl-3 pt-1 pb-1 border ${agents[index].slug === "khoj" ? "border-orange-500" : "border-stone-100"} rounded-md shadow-sm`} onClick={handleAgentsClick(agents[index].slug)}>
```